packages/fx-core/src/plugins/resource/bot/resources/messages.ts

```diff
@@ -42,6 +42,8 @@ export class Messages {
     `Successfully retrieved zip package from ${zipUrl}.`;
   public static readonly FallingBackToUseLocalTemplateZip =
     "Falling back to use local template zip.";
+  public static readonly ResourceProviderExist = (rp: string): string =>
+    `Resource Provider ${rp} already exist, do not need to create.`;
```

Comment:
Actually we don't **create** a resource provider, we just **register** it. May need to refine the wording.